packages/design-system/src/components/web-components/ds-filter-chip/ds-filter-chip.stories.tsx

```diff
@@ -0,0 +1,153 @@
+import type { Meta } from '@storybook/react';
+import { useEffect } from 'react';
+import { action } from '@storybook/addon-actions';
+import WebComponentDocTemplate from '../../../../../../.storybook/docs/WebComponentDocTemplate.mdx';
+import { webComponentDecorator } from '../storybook';
+import './ds-filter-chip';
+
+const meta: Meta = {
+  title: 'Web Components/ds-filter-chip',
+  decorators: [webComponentDecorator],
+  parameters: {
+    docs: {
+      page: WebComponentDocTemplate,
+      description: {
+        component:
+          'For information about how and when to use this component, [refer to its full documentation page](https://design.cms.gov/components/filter-chip/).',
+      },
+      componentEvents: {
+        'ds-delete': {
+          description: 'A callback function triggered when the user clicks the filter chip button.',
+        },
+      },
+    },
+  },
+  args: {
+    label: 'Example Filter Chip',
+    'clear-label': 'Remove',
+  },
+  argTypes: {
+    'clear-label': {
+      description: 'Labels filter action, i.e., "Remove." For screenreader support.',
+      control: 'text',
+    },
+    'class-name': {
+      description: 'CSS class to apply custom styles to the Drawer.',
+      control: 'text',
+    },
+    label: {
+      description: 'Text for the filter chip.',
+      control: 'text',
+    },
+    'root-id': {
+      description:
+        'Id for filter chip button. If not provided, a unique id will be automatically generated and used.',
+      control: 'text',
+    },
+    size: {
+      description:
+        'Sets the size of the chip to larger version. The only acceptable value is "big".',
+      control: 'text',
+    },
+    'use-alternate-icon': {
+      description: 'Use alternate thinner close icon in place of standard.',
+      control: 'boolean',
+    },
```

Comment:
Ask me about this in the office hours session if there's down time

Reply:
Thanks for bringing this up. Closing this loop:
We are creating a ticket to do the discovery work necessary to unify close icons across the design system. This work will happen outside of this pull request.